cmd/operion-source-manager/manager.go

```diff
@@ -287,14 +287,17 @@ func (spm *ProviderManager) updateTriggersWithSourceIDs(ctx context.Context, tri
 	spm.logger.Info("Updating triggers with source IDs", "mapping_count", len(triggerToSourceMap))
 
 	// Get all workflows to find and update triggers
-	workflows, err := spm.persistence.WorkflowRepository().GetAll(ctx)
+	result, err := spm.persistence.WorkflowRepository().ListWorkflows(ctx, persistence.ListWorkflowsOptions{
+		Limit:        1000, // Get all workflows for source ID management
+		IncludeNodes: true, // Need nodes for trigger processing
```

Comment:
We will need to revisit this in the future... The source manager only does this because there was no way to create a complete workflow via endpoints. This was the way found to read the json files and compute the sources to be handled. When all of the endpoints are created, we can remove this.